cmd/cwa/testdata/test.rs

```diff
@@ -0,0 +1,141 @@
+extern crate core;
+use std::str;
+
+//#[link(name = "cwa", wasm_import_module = "cwa")]
+extern "C" {
```

Comment:
The `"C"` should be dropped (see [JavaScript Interoperation](https://rustwasm.github.io/book/reference/js-ffi.html) in the Rust Wasm book).